geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/sortedset/AbstractZAddIntegrationTest.java

```diff
@@ -35,12 +35,17 @@
 import redis.clients.jedis.Protocol;
 import redis.clients.jedis.params.ZAddParams;
 
+import org.apache.geode.redis.ConcurrentLoopingThreads;
 import org.apache.geode.redis.RedisIntegrationTest;
 import org.apache.geode.redis.internal.RedisConstants;
+import org.apache.geode.redis.internal.netty.Coder;
 import org.apache.geode.test.awaitility.GeodeAwaitility;
 
 public abstract class AbstractZAddIntegrationTest implements RedisIntegrationTest {
   private JedisCluster jedis;
```

Comment:
I'd like to see a test for when increment is infinity, and when incrementing a score of infinity, since those are both valid. It would need to test negative and positive infinity, and that you can pass in "Infinity", "+Infinity", "-Infinity", "infinity", "+infinity", "-infinity", "inf", "+inf", and "-inf" as increments.

Reply:
As zadd with incr option internally calls the zincrby methods. I think the test coverage for zincrby should be good enough already.

Reply:
good point 👍